src/enigma2/data/Timer.cpp

```diff
@@ -260,12 +274,47 @@ bool Timer::UpdateFrom(TiXmlElement* timerNode, Channels &channels)
     }
   }
 
+  if (ContainsTag(TAG_FOR_PADDING))
+  {
+    std::string paddingTag = ReadTag(TAG_FOR_PADDING);
```

Comment:
const std::string? Di you need the var at all? Its value is used only one time.

Reply:
Done